### ISSUE TITLE
Software Deployment Dockerization

### ISSUE THREAD

**Issue by krutarth4** [amosproj/amos2023ws04-pipeline-manager#76]
## User story
1. As a SD, I want to deploy the whole system architecture with one line of code.
3. So that rather than executing each docker files separately, it could be done just with one command

## Acceptance criteria
* System is up and running
- [ ] Frontend
- [ ] Backend
- [ ] MongoDB
- [ ] Apache airflow

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been tested
* Feature has been merged into the main branch
* Important parts have been documented in the technology wiki
* All acceptance criteria were met
* Product owner approved features
* Developers agreed to release

**Comment by krutarth4** [comment]
This issue is for whole application deployment using docker compose file

@Elementator @eubec
